### ISSUE TITLE
Duplicate backtracking steps displayed in debugger?

### ISSUE THREAD

**Issue by amandion** [firasdib/Regex101#581]
Here is a simple example: [hello world regex](https://regex101.com/r/gR5pT2/1l)

In the debugger view, I'm confused about steps 5 and 6. I understand why backtracking occurs, but why twice? I would expect step 5 to be the only backtracking step as the literal 'd' in the regex is being tested against the 'd' in the string. I'm sure there is good reason for it but it is a little hard to follow in the debugger.

Thanks.

**Comment by Ouims** [comment]
The correct link: https://regex101.com/r/gR5pT2/1

**Comment by Ouims** [comment]
It backtracks once into the group, and once for the .*

**Comment by firasdib** [comment]
Generally we should avoid using the issues tracker for questions. I recommend you hop on irc instead, it would have taken 3 minutes instead of 3 days to receive an answer :-)
